Sources/StreamChatUI/ChatMessageList/ChatMessage/ChatMessageLayoutOptionsResolver.swift

```diff
@@ -39,6 +39,11 @@ open class _ChatMessageLayoutOptionsResolver<ExtraData: ExtraDataTypes> {
         
         var options: ChatMessageLayoutOptions = []
 
+        // The text should be centered without a bubble for system messages
+        guard message.type != .system else {
+            return [.text, .centered]
+        }
+
```

Comment:
let's have it also covered by the unit-test

Reply:
Thanks! I've added a unit test for this